constants.gradle

```diff
@@ -0,0 +1,62 @@
+ext {
+    // build dependencies
+    gradleBuildToolsVersion = '7.3.1'
```

Comment:
Total nit but I feel like keeping versions separate was a sort of convention I saw in a lot of android projects.

Reply:
Could you elaborate, because specifying versions as constants is exactly what this file is for.

Reply:
As a convention version constants are kept in a separate file called versions I think typically. It looks like you moved them with other constants here.

Reply:
oh i see, ok, i could make a separate constants file for non-version things